sippy-ng/src/componentReadiness/ComponentReadiness.js

```diff
@@ -0,0 +1,596 @@
+import './ComponentReadiness.css'
+import { Alert, TabContext } from '@material-ui/lab'
+import { DateTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers'
+import {
+  Drawer,
+  Grid,
+  TableContainer,
+  Tooltip,
+  Typography,
+} from '@material-ui/core'
+import { Fragment, useEffect } from 'react'
+import { GridToolbarFilterDateUtils } from '../datagrid/GridToolbarFilterDateUtils'
+import { Link, Redirect, Route, Switch, useRouteMatch } from 'react-router-dom'
+import { useTheme } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import CheckBoxList from './CheckboxList'
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft'
+import ChevronRightIcon from '@material-ui/icons/ChevronRight'
+import clsx from 'clsx'
+import EmojiEmotionsOutlinedIcon from '@material-ui/icons/EmojiEmotionsOutlined'
+import FireplaceIcon from '@material-ui/icons/Fireplace'
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline'
+import IconButton from '@material-ui/core/IconButton'
+import MenuIcon from '@material-ui/icons/Menu'
+import PropTypes from 'prop-types'
+import React from 'react'
+import ReleaseSelector from './ReleaseSelector'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+
+function SeverityIcon(props) {
+  const status = props.status
+
+  let icon = ''
+
+  if (status > 8) {
+    icon = (
+      <EmojiEmotionsOutlinedIcon
+        data-icon="EmojiEmotionsOutlinedIcon"
+        fontSize="large"
+        style={{ color: 'green' }}
+      />
+    )
+  } else if (status > 5) {
+    icon = (
+      <FireplaceIcon
+        data-icon="FireplaceIcon"
+        fontSize="small"
+        style={{
+          color: 'yellow',
+        }}
+      />
+    )
+  } else {
+    icon = (
+      <FireplaceIcon
+        data-icon="FireplaceIcon"
+        fontSize="large"
+        style={{ color: 'red' }}
+      />
+    )
+  }
+
+  return <Tooltip title={status}>{icon}</Tooltip>
+}
+
+SeverityIcon.propTypes = {
+  status: PropTypes.number,
+}
+
+function Cell(props) {
+  const status = props.status
+  const theme = useTheme()
+
+  if (status === undefined) {
+    return (
+      <Tooltip title="No data">
+        <TableCell
+          className="cell-result"
+          style={{
+            textAlign: 'center',
+            backgroundColor: theme.palette.text.disabled,
+          }}
+        >
+          <HelpOutlineIcon style={{ color: theme.palette.text.disabled }} />
+        </TableCell>
+      </Tooltip>
+    )
+  } else {
+    return (
+      <TableCell
+        className="cell-result"
+        style={{
+          textAlign: 'center',
+          backgroundColor: 'gray',
```

Comment:
Is it possible to use a white background? Maybe it looks ok if you use the dark variants of the palette colors I shared above

Reply:
it does look better.